api/configs/middleware/__init__.py

```diff
@@ -89,7 +89,7 @@ class DatabaseConfig:
     @computed_field
     @property
     def SQLALCHEMY_DATABASE_URI(self) -> str:
-        db_extras = f"?client_encoding={self.DB_CHARSET}" if self.DB_CHARSET else ""
+        db_extras = f"?{self.DB_EXTRAS}" if self.DB_EXTRAS else ""
```

Comment:
This change will result in a breaking change, and we need to keep the previous DB_CHARSET. Ensure forward compatibility

Reply:
Fixed